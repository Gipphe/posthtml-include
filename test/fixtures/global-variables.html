<div>Should merge global locals</div>
<include src="./test/fixtures/includes/6.html">
  {
  "localVariable": "local"
  }
</include>
