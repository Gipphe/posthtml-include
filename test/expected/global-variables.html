<div>Should merge global locals</div>
<div>local</div>
<div>global</div>
